geometry_msgs/Point32[] pixels_corners
